src/components/Pagination/PrevPaginationItem.tsx

```diff
@@ -30,5 +40,12 @@ export const PrevPaginationItem = withTheme(PrevPaginationItemComponent)
 const Button = styled(ItemButton)`
   ${({ theme }: InjectedProps) => css`
     color: ${theme.palette.TEXT_GREY};
+    &.paginationItem.s.square {
+      font-size: 13px;
+      border-radius: 4px;
+      &:disabled {
+        color: #c1c1c1;
+      }
+    }
   `}
 `
```

Comment:
[IMO] Instead of `color` and `font-size`, use `color` and `size` are provided as Icon Props.
And if `disabled-color` is defined on the ItemButton, there is no need to override the style in this file.